here/README.md

```diff
@@ -115,10 +115,27 @@ For an exploratory description of coverage (or how much probe data there is) for
 
 #### Getting link attributes
 
-The Traffic Analytics `link_dir` is a concatenation of the `streets` layer `link_id` and a `travel direction` character (F,T). `F` and `T` represent "From" and "To" relative to each link's reference node, which is *always* the node with the lowest latitude. In the case of two nodes with equal latitude, the node with the lowest longitude is the reference node. To join `ta` data to the `here_gis.streets_att` table use the following:
+The Traffic Analytics `link_dir` is a concatenation of the `streets` layer `link_id` and a `travel direction` character (F,T). `F` and `T` represent "From" and "To" relative to each link's reference or start node.
+
+The geometries associated with a `link_id` are only given in one direction, so may need to be reversed. To join `link_dir`s to the `streets` table and get the correctly directed link geometries, you may do like:
```

Comment:
I think for getting reversed geometry, we can mention the use of `routing_streets_xx_x` since all necessary geometries get reversed in that table. relevant code in: 
https://github.com/CityofToronto/bdit_data-sources/blob/master/here/traffic/sql/create_here_routing.sql

So to get the correct geom and street attributes we could directly join to routing_streets:
```
SELECT 
    streets.link_dir, 
    attributes.link_id, 
    attributes.st_name, 
    streets.geom
FROM here.routing_streets_22_2 streets
join here_gis.streets_att_22_2 as attributes on left(ta.link_dir,-1)::numeric = attributes .link_id
```

Reply:
Mentioned!

I do like the query as I wrote it too though. `here_gis.streets_xx_x` is the intutive place to look for geometries, and the example makes sense of the whole F/T thing.